client/src/components/Header/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Auth from "../../utils/auth";
+
+const Header = () => {
+  /*for log out kill session*/
+  const logout = (event) => {
+    event.preventDefault();
+    Auth.logout();
+  };
+  return (
+    <header className="bg-secondary mb-4 py-2 flex-row align-center">
+      <div className="container flex-row justify-space-between-lg justify-center align-center">
+        <a href="/">
+          <h1>Events For Me</h1>
+        </a>
+        <nav>
+          {Auth.loggedIn() ? (
+            <>
+              <Link to="dashboard">Dashboard</Link>
+              <Link to="create-event">Create Event</Link>
+              <Link to="my-tickets">My Tickets</Link>
+              <Link to="account">Account Details</Link>
```

Comment:
Currently does not have page content. Need to update Account Details component before merge to main brach.

Reply:
Merged new Account Details.